### DISCUSSION TITLE
Bo

### DISCUSSION THREAD

**Discussion by Bhowell52** [arkypita/LaserGRBL#2370] [General]
I have a longer ray 5 machine and I'm on my 2nd machine with the same problems. While burning my machine just stops and sometimes the laser does not rurn off and once that happens i have lost that project. Have replaced 3 motherboards, 2 lasers, USB cable and have talking yo longer company by email for the last 60 days with no luck fixing problems. Also reinstalled GRBL 3 times.
Longer is in progress of sending me a 3rd machine to try. I had a yorahome home machine for 4 years and never had a problem. I'm needing some good ideas on how to fix this. The machine eill only run about 20 minutes before shutting down.
 Have set my machine to stop every 30 minutes for 2 minutes to cool down and have a fan blowing on the laser.
Any ideas on how to fix this?
Thanks
Bo Howell 
615.481.7007 
Bhowell52bh@gmail.com 
GRBLsays looks like there is a problem exciting this job

**Comment by arkypita** [comment]
Hi, it's hard to say what's wrong with your longer.
I have used both Longer and YoraHome without any issues.

My check list is (in execution order):
- Try with "check" mode (send $C to the machine typing $C in the "type gcode here" box and enter) and then send a job to the machine. The machine receive and immediatly reply ok without executing, so this test the communication only).
- Try with motor cable and laser cable disconnected (lasergrbl send the gcode stream and the board execute them, but no motor electrical noise). Warning: always turn off and remove power to the machine when connecting/disconnect motor cables!
- Try with a different PC.
- Try with a different power supply.
- Disable PC power balance (set power plan to "maximum performance").
- Disable PC USB sleep.

**Reply by Hostaler** [reply]
I think this is the list of command codes like : $C, $H... that Lasergrbl uses, I have not found this listing on the Lasergrbl main page.
If anyone remembers where we have this command page in Lasergrbl, please share it, so we don't have to go looking for it on other pages.Thank you


https://github.com/gnea/grbl/blob/master/doc/markdown/commands.md
